src/hooks/users/useWorldUsersById.ts

```diff
@@ -0,0 +1,41 @@
+import { useMemo } from "react";
+
+import { User } from "types/User";
+
+import { worldUsersByIdSelector } from "utils/selectors";
+
+import { isLoaded } from "hooks/useFirestoreConnect";
+import { useSelector } from "hooks/useSelector";
+
+import { useConnectWorldUsers } from "./useConnectWorldUsers";
+
+/**
+ * @debt typing, Record implies that a User will exist for literally any given string, which is untrue
+ * @deprecated use useWorldUsersByIdWorkaround until we refactor this to fix the typing issue across the codebase
+ */
+export const useWorldUsersById = () => {
```

Comment:
[note] This hook was split out of `src/hooks/users/index.ts` into it's own file here.